test/unit/commandLineParser.spec.ts

```diff
@@ -1,235 +1,177 @@
-import { findConfigFile, parseCommandLine, parseTsConfigString } from "../../src/CommandLineParser";
-import { LuaLibImportKind, LuaTarget } from "../../src/CompilerOptions";
-
-test.each([
-    { args: [""], expected: LuaLibImportKind.Inline },
-    { args: ["--luaLibImport", "none"], expected: LuaLibImportKind.None },
-    { args: ["--luaLibImport", "always"], expected: LuaLibImportKind.Always },
-    { args: ["--luaLibImport", "inline"], expected: LuaLibImportKind.Inline },
-    { args: ["--luaLibImport", "require"], expected: LuaLibImportKind.Require },
-])("CLI parser luaLibImportKind (%p)", ({ args, expected }) => {
-    const result = parseCommandLine(args);
-    if (result.isValid === true) {
-        expect(result.result.options.luaLibImport).toBe(expected);
-    } else {
-        expect(result.isValid).toBeTruthy();
-    }
-});
+import * as ts from "typescript";
+import * as tstl from "../../src";
 
-test("CLI parser invalid luaLibImportKind", () => {
-    const result = parseCommandLine(["--luaLibImport", "invalid"]);
-    expect(result.isValid).toBe(false);
-});
+describe("command line", () => {
+    test("should support aliases", () => {
+        const full = tstl.parseCommandLine(["--luaTarget", "5.1"]);
+        const alias = tstl.parseCommandLine(["-lt", "5.1"]);
+        expect(full).toEqual(alias);
+    });
+
+    test("should support standard typescript options", () => {
+        const commandLine = "--project tsconfig.json --noHeader -t es3 -lt 5.3";
+        const result = tstl.parseCommandLine(commandLine.split(" "));
+
+        expect(result.errors).not.toHaveDiagnostics();
+        expect(result.options).toEqual({
+            project: "tsconfig.json",
+            noHeader: true,
+            target: ts.ScriptTarget.ES3,
+            luaTarget: tstl.LuaTarget.Lua53,
+        });
+    });
 
-test.each([
-    { args: [""], expected: LuaTarget.LuaJIT },
-    { args: ["--luaTarget", "5.1"], expected: LuaTarget.Lua51 },
-    { args: ["--luaTarget", "5.2"], expected: LuaTarget.Lua52 },
-    { args: ["--luaTarget", "jit"], expected: LuaTarget.LuaJIT },
-    { args: ["--luaTarget", "JIT"], expected: LuaTarget.LuaJIT },
-    { args: ["--luaTarget", "5.3"], expected: LuaTarget.Lua53 },
```

Comment:
I think actually having these tests was very nice, since they directly specify our interface, I don't think they should be removed.